Assets/HoloToolkit/Sharing/Scripts/Utilities/SharingServiceStatus.cs

```diff
@@ -0,0 +1,61 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+namespace HoloToolkit.Sharing
+{
+    public class SharingServiceStatus : MonoBehaviour
+    {
+        enum Status
+        {
+            Disconnected,
+            Connecting,
+            Connected,
+        }
+
+
+        [Tooltip("Reference to Text UI control where the Status info should be displayed.")]
+        public Text Text;
+
+        [Tooltip("Reference to Image UI control where the Status background color should be applied.")]
+        public Image Background;
+
+        void Start()
+        {
+            SetStatus(Status.Disconnected);
+
+            SharingStage.Instance.SharingManagerConnected += SharingManagerConnected;
+
+            if (SharingStage.Instance.AutoDiscoverServer)
+                SetStatus(Status.Connecting);
```

Comment:
Haha NIT is short for nitpicking :) basically a recommendation style comment.
http://www.dictionary.com/browse/nitpicking